### DISCUSSION TITLE
RP Metadata Not Updating

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#322] [Fixed Issues]
@rern 

Since 20210507, the Radio Paradise metadata does not update without the browser being manually refreshed.

**Comment by klassikmann** [comment]
Perhaps correlated: A local "Update database" shows the same effect, blinking until refresh.

**EDIT 11.05.21**:
Issue regarding "Update database" vanished after full system upgrade (pacman -Syu). I don't know why.

**Comment by R101G** [comment]
Fixed by 20210515 update, thanks.
